src/hotspot/share/runtime/threads.cpp

```diff
@@ -1328,6 +1328,17 @@ void Threads::print_on(outputStream* st, bool print_stacks,
         p->trace_stack();
       } else {
         p->print_stack_on(st);
+        oop thread_oop = p->threadObj();
+        if (thread_oop != nullptr) {
+          if (p->is_vthread_mounted()) {
+            oop vt = p->vthread();
+            assert(vt != nullptr, "");
+            st->print_cr("   \tMounted virtual thread #" INT64_FORMAT, (int64_t)java_lang_Thread::thread_id(vt));
```

Comment:
Please no manual indentation, see remarks above.

Reply:
I'm leaving minimal indentation and I will update the code on a new PR to rely on your changes if it's OK for you @tstuefe